packages/web-components/src/progress-bar/progress-bar.styles.ts

```diff
@@ -13,7 +13,7 @@ import {
   colorPaletteRedBackground3,
 } from '../theme/design-tokens.js';
 
-/** Text styles
+/** ProgressBar styles
```

Comment:
I'm fine if we keep this, but ideally we can handle separate concerns in separate PR's as a policy

Reply:
Yeah, it was super small so I kept it in. But I understand.